CadRevealFbxProvider.Tests/FbxProviderAttributeParserTests.cs

```diff
@@ -37,6 +36,13 @@ public void ParseCorrectAttributesTest()
         );
     }
 
+    [Test]
+    public void ParseCorrectAttributesTest()
+    {
+        ParseCorrectAttributesTest("\\fbx_test_model.csv");
+        ParseCorrectAttributesTest("\\fbx_test_model_with_header_on_row_two.csv");
+    }
```

Comment:
If you want to you can use a special test setup (TestCase) that will create a new test based on the parameters given instead of this wrapper thing.

https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcase.html